### DISCUSSION TITLE
First-time contribution

### DISCUSSION THREAD

**Discussion by liquid-dragons** [nushell/nushell#13940] [Q&A]
Hi, I ran into https://github.com/nushell/nushell/issues/11215 the other day, and Im considering having a look at it and opening a PR to fix that - is there anything I should be aware of, or that might be helpful?

So far my first impression is that since a table is represented as a list of records (iiuc), we would 'just' need to distinguish 
'some records lack this name' from 'no records contain this name' (in `follow_cell_path` perhaps?), and return a new error. Does that make sense?

Thanks in advance!

**Comment by fdncred** [comment]
Makes sense to me. It would be nice to say something like, "Row 5 is missing the <col_name> column". Thanks for trying to make nushell better.
